include/ProxyWifi/ProxyWifiService.hpp

```diff
@@ -50,19 +50,11 @@ class ProxyWifiService
     virtual void Stop() = 0;
 };
 
-/// @brief Indicate who is concerned by a connection/disconnection event
-enum class EventSource
+/// @brief Indicate the status of a guest initiated operation
+enum class OperationStatus
 {
-    Host,  ///< The host was (dis)connected
-    Guest, ///< The guest was (dis)connected
-};
-
-/// @brief Indicate the status of a guest initiated connection
-enum class GuestConnectStatus
-{
-    Starting, ///< A connection request was received and will result in trying to connect host interfaces
-    Succeeded, ///< An host interface has been connected to the requested network and the guest will be notified of the success
-    Failed ///< No host interface could be connected to the requested network and the guest will be notified of the failure
+    Succeeded,
+    Failed
```

Comment:
nit: 
```suggestion
    Failed,
```

Reply:
Other enums don't have a trailing comma, this was a style issue before.

Reply:
Ok, I thought I saw other ones that did. Either way, just style.